test/nuts/suggestType/testProj/force-app/main/default/layouts/MyTestObject__c-MyTestObject Layout.Layout-meta.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd like to have a test that "emailService still correctly resolves" since we don't have coverage for that (that I know of!) and it's part of the change.

Reply:
Added a test 👍